test/mailers/receipt_mailer_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
+class ReceiptMailerTest < ActionMailer::TestCase
+  def test_receipt_email
+    thesis = theses(:one)
+    email = ReceiptMailer.receipt_email(thesis)
+
+    # Send the email, then test that it got queued
+    assert_emails 1 do
+      email.deliver_now
```

Comment:
Do we clean up the email-delivery location after the test?

Reply:
No, testing of emails uses an in memory array.

Reply:
Sweet.